web/app.js

```diff
@@ -1655,7 +1655,8 @@ const PDFViewerApplication = {
           baseURL: this.baseUrl,
           filesize: this._contentLength,
           filename: this._docFilename,
-          metadata: this.metadata,
+          metadata: this.rawMetadata,
```

Comment:
With the suggested changes in `src/display/api.js` and `src/display/metadata.js`, this needs to be
```suggestion
          metadata: this.metadata?.getRaw(),
```